docs/src/docs/asciidoc/examples/src/main/java/sample/jpa/service/authorization/JpaOAuth2AuthorizationService.java

```diff
@@ -88,7 +89,7 @@ public OAuth2Authorization findByToken(String token, OAuth2TokenType tokenType)
 
 		Optional<Authorization> result;
 		if (tokenType == null) {
```

Comment:
I agree that it could improve readability, however, I also think the existing code is fine.

Also, code improvements like this should go in a separate PR as it's not related to OIDC Logout. We only make changes that are applicable and needed for the PR.